app/components/elements/FontStyles.js

```diff
@@ -205,9 +205,10 @@ export const TopNavLink = Styled(NavLink)`
 
 export const TextLink = Styled(Link)`
   color: ${colors.text.link};
-  text-decoration: none;
-  line-height: ${lineHeights[0]};
+  font-size: ${fontSizes[1]}px;
   font-family: ${fonts.default};
+  line-height: ${lineHeights[0]};
+  text-decoration: none;
   &:hover {
```

Comment:
Are the hover styles actually required here, since they don't overwrite any of the default styles?